gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Comment:
Why did this change from `-all` to `-bin`?

Reply:
Okay, lets wait till -all is out. I remember we need all for some reason.

Reply:
@ArjunaKumarMohanta - This is a major upgrade of Gradle. Can I ask why the hurry? Or what you need in this upgrade as it correlates to Apktool?